src/pages/contact.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 
 import { Layout, Link, Form } from '@components';
-import { encode } from '@utils/encode';
```

Comment:
Ooh nice - since we're not using this now we could delete the util (and test file) 🎉